rpm-build/SPECS/securedrop-workstation-dom0-config.spec

```diff
@@ -123,6 +123,10 @@ find /srv/salt -maxdepth 1 -type f -iname '*.top' \
     | sed -e 's/\.top$$//g' \
     | xargs qubesctl top.enable > /dev/null
 
+# Force full run of all Salt states
```

Comment:
In the past this has only been committed to the release branch - if it lands in main it'll also show up in the nightlies, etc.

Reply:
yup, adding a commit to suppress it for the main merge (IMO should not require retesting, and testing will happen in RCs immediately anyway)